src/device-registry/bin/www

```diff
@@ -131,15 +131,31 @@ const run = async() => {
                         logObject("error.details[0].message", error.details[0].message)
                         logObject("error.details[0]", error.details[0])
                         logObject("error.details", error.details)
-                        logger.error(
-                            `KAFKA: ALL the input validation errors --- ${JSON.stringify(error.details)}`
-                        );
-
-                        logger.info(
-                            `KAFKA: the VALUE for ALL the shared input validation errors --- ${JSON.stringify(value)}`
-                        );
-
-
+                        const errorDetails = error.details.map(detail => {
```

Comment:
looping through the KAFKA input validation errors so that we pick out details which will be beneficial to the Hardware Team while troubleshooting. @deogratius2phantom @BenjaminSsempala